packages/template-react/src/home/index.js

```diff
@@ -8,7 +8,7 @@ export default function Home() {
       <Helmet>
         <title>Hops Demo</title>
       </Helmet>
-      <h1 className={styles.headline}>Hello World!</h1>
+      <h1 className={styles.headline}>Hello World! {process.env.FOO_ROBIN}</h1>
```

Comment:
I do not really like the idea of using these global variables directly. Instead I would suggest making them available using a HOC.

Reply:
The issue with using a HoC is that you can't really access environment variables in the mixin methods of a runtime mixin. At least not when it is being executed on the browser.
So if you need access to environment variables in, for example, your `bootstrap()` method, you're out of luck when using a HoC.
We had the idea of setting `global._hopsEnvironmentVariables` in the core mixin and passing that into `enhanceData`, so that you could access environment variables in your mixins through `global._hopsEnvironmentVariables`, but that felt unintuitive.
That's how we arrived at the idea of using webpack to alias the `process.env` namespace. Therefore you could access environment variables in all classes of mixins in the same way.

Reply:
How about introducing a HopsMixin base class that makes those additional variables available as instance props on its constructor?